bin/merge

```diff
@@ -49,6 +49,8 @@ if [[ "$merged" == "false" ]]; then
     cd $directory
     git checkout -b $tmp_branch $base_sha
     git pull --ff-only $pr_clone $pr_ref
+    git checkout $base_ref
+    git merge --ff-only -m "$(printf "Merge branch '%s'\n\nCloses #%s." "$pr_ref" "$pull_request")" $tmp_branch
```

Comment:
I would replace the `-m "$(printf "Merge branch '%s'\n\nCloses #%s." "$pr_ref" "$pull_request")"` with `--no-edit` or perhaps a different message (I forget what the auto-generated message looks like). The particular message there was done to trigger auto-closing of the GitHub PR, but that’s no longer necessary, since it’s a clean merge now.